django/db/models/query.py

```diff
@@ -647,6 +647,7 @@ def create(self, **kwargs):
         """
         obj = self.model(**kwargs)
         self._for_write = True
+        obj.for_create = True
         obj.save(force_insert=True, using=self.db)
```

Comment:
Have you considered changing the signature of `force_insert` to either allow passing a sentinel object to denote that only the current class should be inserted or maybe event a set of bases that should be forced to be inserted.

That would basically mean that `force_insert` signature could be changed from `bool` to `bool | set[Type[Model]]` and since a non empty set is still truthy we should maintain backward compatibilty to a reasonable extent.

Today `force_insert=True` means only force insert the current model, attempting to change its meaning to force insert all parents seems backward incompatible and the crux of the problem you are facing.

Maybe what we need to do here is
1. Allow bases to provided to `force_insert` as detailed above
2. Raise a deprecation warning when `force_insert=True` is used and MTI is involved to denote that it will mean force insert every bases in the future
3. Have `force_insert=True` default to `{self.__class__}` during the deprecation period.

Reply:
> 3. Have `force_insert=True` default to `{self.__class__}` during the deprecation period.

I think having `force_insert=False` will be a more appropriate default value here, as the current behavior is to not `force_insert` the parent when create is called.